### PR TITLE
Keep *.info.cleaned. Closes #10

### LINKED ISSUE #10: Keep *.info.cleaned for processing
https://github.com/mikeferguson/code_coverage/blob/22fc9fd0e96f11d693aedb6a26c4d94d161cd2c9/cmake/Modules/CodeCoverage.cmake#L147

Directly deletes the *.info.cleaned after creating the html report.
However I would like to keep the cleaned tracefile for further processing and maybe some others do as well.

Open for discussion!

### PR DESCRIPTION
**Pull request by agutenkunst**
This way the file gets overwritten at the next run. Should be fine...

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,10 @@
 Changelog for package code_coverage
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Forthcoming
+-----------
+* Keep *.info.cleaned tracefile
+
```
Comment (mikeferguson) [inline CHANGELOG.rst]: please revert the changes to changelog - this gets auto generated when I do a release

**Review by mikeferguson — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (agutenkunst) [reply CHANGELOG.rst]: Done. Sry.

**Comment by agutenkunst** [comment]
Could you please release on melodic and kinetic :star:

**Comment by agutenkunst** [comment]
Friendly ping :speech_balloon:

**Comment by ct2034** [comment]
Another friendly ping ;-) @mikeferguson

**Comment by agutenkunst** [comment]
Hi @mikeferguson this is a bit of a blocker for our plan to integrate our coverage checks with the Travis CI.
We definitely would owe you a :beer: or :beer: :beer: if you can help and I am sure a @ct2034 always pays its debts :smile:

**Comment by mikeferguson** [comment]
Sorry about the delays -- I'll cut a new release this afternoon
